bricks/vs/src/code-editor/utils/isInEmbedded.ts

```
@@ -0,0 +1,69 @@
+}
+
+/**
+ * Find the next match of the word in the model, but no loops to the beginning.
+ */
+function findNextMatch(
+  model: monaco.editor.ITextModel,
+  word: string,
+  position: monaco.Position
+) {
+  const match = model.findNextMatch(word, position, false, false, null, false);
+  if (match && match.range.getEndPosition().isBefore(position)) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)